packages/react-devtools-inline/playwright.config.js

```diff
@@ -18,9 +18,10 @@ const config = {
     react_version: process.env.REACT_VERSION
       ? semver.coerce(process.env.REACT_VERSION).version
       : reactVersion,
+    trace: 'retain-on-failure',
+    // Some of our e2e tests can be flaky. Retry tests to make sure the error isn't transient
+    retries: 3,
   },
-  // Some of our e2e tests can be flaky. Retry tests to make sure the error isn't transient
-  retries: 2,
```

Comment:
Does this mean it wasn't retrying before?

Reply:
We were, just twice instead of 3 times

Reply:
I think flaky tests might actually be categorized as an error though, so looking into that as well. Figured we probably want snapshots of failing tests regardless though

Reply:
small nit: was the move of the comment intended?